### PR TITLE
clamp raster to z0

### PR DESCRIPTION
**Pull request by mapsam**
Clamps all raster inputs to z0 no matter what.

Also adds a try/catch to the extent forward projection in lib/shape.js #175 

cc @GretaCB @millzpaugh @springmeyer @samfader

### CODE REVIEW

File: lib/shape.js [hunk 1]
```diff
@@ -96,7 +96,11 @@ Shape.prototype.getExtent = function(callback) {
 
     if (current !== wgs84) {
       transform = new mapnik.ProjTransform(current, wgs84);
-      extent = transform.forward(extent);
+      try {
+        extent = transform.forward(extent);
```
Comment (GretaCB) [inline lib/shape.js]: Two birds! 🎉  🔑  🕊

Comment (mapsam) [reply lib/shape.js]: 🐦 🐤

**Comment by mapsam** [comment]
Per discussion with @GretaCB - we'll plan on releasing this (and other future zoom updates) as a minor release. Incoming 8.5.0 🚀

**Comment by mapsam** [comment]
@springmeyer just saw that you did this over in #173, apologies for not seeing that. Going to go ahead with this PR since it has a try/catch addition as well. 💥

**Pull request merged**